README.md

```diff
@@ -9,31 +9,17 @@ Being part of the Fediverse allows you to be free from data-harvesting corporati
-|                                                                                         *Notifications menu and private messages counter, standard browser on tablet.*                                                                                          |
-|                                                                ![Frio theme in desktop browser](images/screenshots/friendica-frio-brown-profile-2.png?raw=true "Frio theme in desktop browser")                                                                 |
-|                                                                                                         *Number of visible contacts, standard browser.*                                                                                                         |
-|                                                                ![Frio theme in desktop browser](images/screenshots/friendica-frio-brown-profile-1.png?raw=true "Frio theme in desktop browser")                                                                 |
-|                                                                                                   *Network posts chronologically ordered, standard browser.*                                                                                                    |
-|                                                                    ![Vier theme in desktop browser](images/screenshots/friendica-vier-profile.png?raw=true "Vier theme in desktop browser")                                                                     |
-|                                                                                                      *Vier theme, desktop browser. Public timeline view.*                                                                                                       |
-|                                                                   ![Vier theme in desktop browser](images/screenshots/friendica-vier-community.png?raw=true "Vier theme in desktop browser")                                                                    |
-|                                                                                                    *Vier theme, desktop browser. Community post displayed.*                                                                                                     |
+### Friendica on desktop
+
+![Frio theme in desktop browser](images/screenshots/friendica-2022-12-frio-desktop.png?raw=true "Frio theme in desktop browser")
+
+### Friendica on mobile
+
+![Frio theme in mobile browser](images/screenshots/friendica-2023-10-frio-mobile-timeline-dark-blue.png?raw=true "Frio theme in mobile browser") ![Frio theme in mobile browser](images/screenshots/friendica-2023-10-frio-mobile-options-light-blue.png?raw=true "Frio theme in mobile browser")
+
+### Alternative Theme "Vier"
+
+![Frio theme in desktop browser](images/screenshots/friendica-vier-community.png?raw=true "Vier theme in desktop browser")
```

Comment:
Is it vier or Frio?

Reply:
This is a keepsake from the previous screenshots, so it is indeed vier.